swagger.yaml

```diff
@@ -994,6 +994,24 @@ definitions:
         $ref: '#/definitions/EditionLinks'
       state:
         $ref: '#/definitions/State'
+      current:
```

Comment:
As far as I understand it, the way we've documented the dataset response is just to return the public response, and not show any concept of current/next in the spec. We should be consistent with the documentation (and if I've misunderstood it and it's already consistent ignore me 👍 )

Reply:
@eldeal , i've set it up returning:    parent{current:foo, next:foo} for auth'd and just parent{current:foo} for non auth'd.

do I need to hide the current concept?